src/index.html

```diff
@@ -2,12 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Product image"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="group-1">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="review-count">Reviews: 5</div>
+      </div>
+
+      <div class="price-container">
+        <div class="price-label">Price:</div>
+        <div class="price">$2,199</div>
+      </div>
```

Comment:
1. you should use only two BEM blocks (card and stars)
2. use semantic tags for text elements
```suggestion
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="card__reviews">Reviews: 5</p>
      </div>

      <div class="card__price">
        <p class="card__price-text">Price:</p>
        <p class="card__price-value">$2,199</p>
      </div>
```